Cargo.toml

```diff
@@ -18,8 +18,11 @@ regex = "1.3.6"
 lazy_static = "1.4.0"
 humantime = "2.0.0"
 log = "0.4.8"
+actix-web = "2.0"
+derive_more = "0.99"
```

Comment:
Macro crates are usually also a compile time oof, but I think the usage of this one is more or less justified?

Reply:
I mean, it's really not that hard to `impl fmt::Display` and match on these and make a real implementation by hand, but I can see how this is demonstrably easier to keep up with; I suppose the question is, is the longer compile time for everything downstream from this core crate less of a devtime cost than being able to easily impl this?

Reply:
This is fodder for later commits and pull requests.